include/invert_laplace.hxx

```diff
@@ -139,7 +139,7 @@ public:
   
   virtual void setCoefEz(const Field2D &val) = 0;
   virtual void setCoefEz(const Field3D &val) { setCoefEz(DC(val)); }
-  virtual void setCoefEz(BoutReal r) { Field2D f(r); setCoefD(f); }
+  virtual void setCoefEz(BoutReal r) { Field2D f(r); setCoefEz(f); }
```

Comment:
Is this a bug fix? If so might be worth noting this more explicitly in the PR description in case we need to port backwards.